share/static/html/spawn.html

```diff
@@ -67,6 +69,20 @@ <h1 style="font-size: 55px">everware</h1>
 require(["jquery"], function ($) {
   // add bootstrap form-control class to inputs
   $("#spawn_form").find("input, select, textarea, button").addClass("form-control");
+  function showError(message) {
+    $("#error_text").text(message);
+    $("#error_text").show();
+  }
+  $("#spawn_form").on('submit', function (event) {
+    var repo_url = $('#repository_input').val();
+    if (!repo_url) {
+      showError('You have to provide the URL to a git repository.');
+      return false;
+    } else if (repo_url.lastIndexOf('git://', 0) == 0) {
+      showError("git protocol isn't supported yet");
```

Comment:
"git protocol is not supported yet. Please use the HTTPS URL scheme"? Basically something that tells the user what they should do instead.